Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3307,10 +3308,68 @@ def handle_msg(file_path, file_name, parse_only_headers=False, max_depth=3):
     if not msg:
         raise Exception("Could not parse msg file!")
 
-    email_data = msg.as_dict(max_depth)
+    msg_dict = msg.as_dict(max_depth)
+
+    try:
+        format_string = msg_dict['Headers'].split('Content-type:')[1].split(';')[0]
+    except ValueError:
+        format_string = ''
+    except IndexError:
+        format_string = ''
+
+    headers = []
+    headers_map = dict()  # type: dict
+    header_key = 'initial key'
+    header_value = 'initial header'
+
+    for header in msg_dict['Headers'].split('\n'):
+        if len(header) > 0 and not header == ' ' and 'From nobody' not in header:
+            if not header[0] == ' ' and not header[0] == '\t':
+                if header_value != 'initial header':
+                    headers.append(
+                        {
+                            'name': header_key,
+                            'value': header_value
+                        }
+                    )
+
+                    if header_key in headers_map:
+                        # in case there is already such header
+                        # then add that header value to value array
+                        if not isinstance(headers_map[header_key], list):
+                            # convert the existing value to array
+                            headers_map[header_key] = [headers_map[header_key]]
+
+                        # add the new value to the value array
+                        headers_map[header_key].append(header_value)
+                    else:
+                        headers_map[header_key] = header_value
+
+                splitted = header.split(' ')
```

Comment:
please improve naming for readability, with a name representing the actual value and purpose of the variable